OtripleS.Web.Api/Models/Users/User.cs

```diff
@@ -0,0 +1,19 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using Microsoft.AspNetCore.Identity;
+
+namespace OtripleS.Web.Api.Models.Users
+{
+    public class User : IdentityUser
+    {
+        public string Name { get; set; }
+        public string FamilyName { get; set; }
+        public UserStatus Status { get; set; }
+        public DateTimeOffset CreatedDate { get; set; }
+        public DateTimeOffset UpdatedDate { get; set; }
+    }
```

Comment:
You can probably do this:
```csharp 
public class User : IdentityUser<Guid>
    {
        public override string UserName
        {
            get => base.Email;
            set => base.Email = value;
        }
    }

```
Along with the rest of the properties that you have.

Reply:
@hassanhabib resolved, thank you